File: src/main/kotlin/inflearn/java/ch01_string/_1_1_문자찾기.java
package inflearn.java.ch01_string;

import java.util.Scanner;

public class _1_1_문자찾기 {
    public static int solution(String str, char t) {
        int answer = 0;

        str = str.toUpperCase();
        t = Character.toUpperCase(t);

        for(char ch: str.toCharArray()) {
            if(ch == t)
                answer++;
        }

        return answer;
    }

    public static void main(String[] args) {
        Scanner kb = new Scanner(System.in);
        String str = kb.next();
        char c = kb.next().charAt(0);

        System.out.print(solution(str, c));
    }
}

File: src/main/kotlin/inflearn/kotlin/ch01/1_1 문자찾기.kt
package inflearn.kotlin.ch01

import java.util.*

class `1_1 문자찾기` {

    private fun solution(str: String, t: Char): Int {
        var answer = 0

        val str = str.uppercase()
        val t = t.uppercaseChar()
        for (ch in str.toCharArray()) {
            if (ch == t) answer++
        }
        return answer
    }

    fun main(args: Array<String>) {
        val kb = Scanner(System.`in`)
        val str = kb.next()
        val c = kb.next()[0]
        print(solution(str, c))
    }
}